packages/react-charting/src/components/TreeChart/TreeChart.types.ts

```diff
@@ -0,0 +1,43 @@
+import { IStyle, ITheme } from '@fluentui/react/lib/Styling';
+import { IStyleFunctionOrObject } from '@fluentui/react/lib/Utilities';
+
+export interface ITreeChartDataPoint {
+  name: string;
+  subname: string;
+  fill: string;
+  children?: Array<ITreeChartDataPoint>;
+}
+export interface ITreeProps {
+  treeData: ITreeChartDataPoint;
+  composition?: number | undefined;
+  width?: number;
+  height?: number;
```

Comment:
did you remove margin from props

Reply:
Yes, I defined it in the main class component, should I include it here again in the props?

Reply:
yes

Reply:
fixed